protocol_connect.go

```diff
@@ -887,13 +889,14 @@ func (m *connectUnaryMarshaler) write(data []byte) *Error {
 
 type connectUnaryRequestMarshaler struct {
 	connectUnaryMarshaler
+	getPolicy      GetPolicy
 	getURLMaxBytes int
 	stableCodec    stableCodec
 	duplexCall     *duplexHTTPCall
 }
 
 func (m *connectUnaryRequestMarshaler) Marshal(message any) *Error {
-	if m.stableCodec != nil && m.getURLMaxBytes > 0 {
+	if m.stableCodec != nil && m.getPolicy != GetPolicyNever {
```

Comment:
Would it be better for us to return an error if the user has `GetPolicyEnforce` set and no stable codec? Otherwise we're falling back to POST, which seems bad. `CodeInternal` seems like the right severity level for misconfiguration like that.